sklearn/datasets/tests/test_common.py

```diff
@@ -0,0 +1,10 @@
+"""Test loaders for common functionality.
+"""
+from sklearn.utils.testing import assert_array_equal
+from sklearn.utils.testing import assert_true
+
+
+def check_return_X_y(bunch, X_y_tuple):
+    assert_true(isinstance(X_y_tuple, tuple))
```

Comment:
We are trying to phase out these assertion functions. Please use a bare assert

Reply:
Also fixed in https://github.com/scikit-learn/scikit-learn/pull/10774/commits/5f73583c99c773fbc204b63afe2acce793fe58fa